### PR TITLE
Connect ingest and phylogenetic workflows to follow pathogen-repo-guide

### PR DESCRIPTION
**Pull request by kimandrews**
## Description of proposed changes

Connect ingest and phylogenetic workflows, following the pathogen-repo-guide:
* Upload ingest output to S3
* Download ingest output from S3 to phylogenetic directory
* Use accession column as ID column for phylogenetic analysis (addresses this github issue: #11)
* Add color scheme matching the new region name format

This PR focuses on making changes that are necessary to move forward with creating a locus-specific workflow in a later PR.

## Related issue(s)
#11 

## Checklist

- [ ] Checks pass

### CODE REVIEW

**Review by jameshadfield — COMMENTED** [review]
Thanks @kimandrews!

I skimmed through the code changes and they look alright, but didn't review in depth. Thanks for uploading the new metadata/sequences to S3.

I ran through the phylogenetic workflow and it works as expected. The steps to upload the dataset to `s3://nextstrain-data` (i.e. to make it live) aren't yet here, but with the new ingest & phylogenetics being part of the master branch I think the new dataset should be made live. You could do this manually now and add the code in a later PR, but we should try to keep the pipeline in-sync with the live dataset.

Auspice flags the following error, which is not part of your PR but should be fixed at some point:

> [Genome annotation] V has length 899 which is not a multiple of 3

An aside I noticed here: We should allow `augur tree` and `augur align` to run with more than one thread. You can see an example of how to connect snakemake and augur's `--nthreads` argument [here](https://github.com/nextstrain/hbv/blob/e3620055a2fdfa9f4196df6af1de8000ae321241/phylogenetic/Snakefile#L75-L88).

File: phylogenetic/example_data/metadata.tsv
[the remark this answers is not in the document]
Comment (kimandrews) [reply phylogenetic/example_data/metadata.tsv:1]: Reworded in [0b90b00](https://github.com/nextstrain/measles/pull/19/commits/0b90b0034f208b7e1cca17da86aa4fbf3b94e204)

File: phylogenetic/defaults/config.yaml [hunk 1]
```diff
@@ -8,9 +8,13 @@ filter:
...
+    strain_id: "accession"
 refine:
     coalescent: "opt"
     date_inference: "marginal"
     clock_filter_iqd: 4
+    strain_id: "accession"    
 ancestral:
     inference: "joint"
+export:
+    strain_id: "accession"
```
[the remark this answers is not in the document]
Comment (kimandrews) [reply phylogenetic/defaults/config.yaml]: Made these changes in [0b3e4c1](https://github.com/nextstrain/measles/pull/19/commits/0b3e4c160e999b3c0c820da0cc66d17b34b1d5ce)

[the remark this answers is not in the document]
Comment (joverlee521) [reply phylogenetic/defaults/config.yaml]: Looks good to me! 

(This [compare view](https://github.com/nextstrain/measles/compare/5390f10a59e47a2a699649d00b89fecc086e82ba..9e096472625c7d7f7f7297ed3a87e7f7c2ee1fd2) shows the diff between your original commit and the force-pushed commit).

**Pull request merged**